hack/generate-configmap.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+# This script generates configmap/s set by arguments
+#
+# To create a CI configmap use the "--ci" option
+# To create an RHEL configmap use the "--rhel" option
+# To create both configmaps use the "--all" option or pass no arguments
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+# Call the function teardown() on the exit of the script
+trap teardown EXIT
+
+teardown() {
+	rm -rf "${TEMP_DIR}"
+}
+
+TEMP_DIR=$(mktemp -d -t keys-XXXXXXXX)
+CONFIGMAP_FILENAME="0000_90_cluster-update-keys_configmap.yaml"
+
+# Generates a public key specified by parameters in the TEMP_DIR directory
+#
+# $1 - Name of the key which will be generated
+# $2 - Verifier public keys separated by internal field separator
+#
+generate_public_key() {
+	truncate -s 0 "${TEMP_DIR}/key.gpg" # Empty the ${TEMP_DIR}key.gpg file
+	for key in ${2}; do
+		gpg --dearmor <"${key}" >>"${TEMP_DIR}/key.gpg"
+	done
+	gpg --enarmor <"${TEMP_DIR}/key.gpg" >"${TEMP_DIR}/${1}"
+	sed -i 's/ARMORED FILE/PUBLIC KEY BLOCK/' "${TEMP_DIR}/${1}"
+}
+
+# Generates a configmap specified by parameters
+#
+# $1 - Name of a generated key in the TEMP_DIR directory
+# $2 - Manifests folder
+# $3 - Store files separated by internal field separator
+#
+generate_configmap() {
+	stores=""
+	for store in ${3}; do
+		stores="--from-file=${store} ${stores}"
+	done
+	read -r -a stores <<<"$stores" # split string into an array
```

Comment:
nit: you can avoid the here string by building `stores` as an array (hooray Bash :):

```bash
stores=()
for store in ...; do
  stores+=("--from-file=${store}")
done
```

Reply:
Fixed in d8197c0.